pandas/core/ops.py

```diff
@@ -343,50 +343,93 @@ def _get_op_name(op, special):
 # -----------------------------------------------------------------------------
 # Docstring Generation and Templates
 
+_add_example_FRAME = """
+>>> a = pd.DataFrame([1, 1, 1, np.nan], index=['a', 'b', 'c', 'd'],
+                     columns=['one'])
+>>> a
+   one
+a  1.0
+b  1.0
+c  1.0
+d  NaN
+>>> b = pd.DataFrame(dict(one=[1, np.nan, 1, np.nan],
```

Comment:
blank line between ases

Reply:
@jreback I am about to put in a PR for another one of the examples.  Where should there be a blank line? Between what would be "cell" executions?

Reply:
@jreback I am about to put in a PR for one of the examples.  Where should there be a blank line? Between what would be "cell" executions?

Reply:
@kurtiskerstein I think there should be only blank lines around "blocks" of code that you want to have appear together in one code block